Add solution to Gigsecond exercise

carlotxra opened the pull request:
Used *exercism extension method on DateAndTime to test if possible to use it.

macta commented:
Extensions do t work yet - we are testing  the move from tags to full packages which makes it possible (probably 1 week away)

carlotxra commented:
Ok no problem, thanks for letting me know. It will be helpful to be able to use extension methods in some of the exercises. Anyway I updated the code and removed it for now.

macta commented:
Yes - we want extension methods hence the move to full packages now we’ve understood the Pharo direction on tags can projects. Short story - we need full packages which creates a different pain for all our exercises but one I’ve midtly resolved now - just need some testing. So thanks for working around it for now

bencoman commented:
@carlotxra We now have extension methods working by moving exercises from package-tags to full-packages - but this means this PR can't be directly merged.  Would it be possible for you to redo it based off current master branch?

Bajger commented:
@carlotxra Solution moved to new branch (easier to merge): https://github.com/exercism/pharo-smalltalk/pull/660
